src/components/FullDetailView/AbstractFullDetailsView.js

```diff
@@ -33,7 +35,8 @@ export class AbstractFullDetailsView extends Component {
       <Modal
         closable={false}
         footer={null}
-        centered
+        // if it's mobile do not center the Modal
+        centered={!isMobile}
         destroyOnClose={true}
         visible={visible}
         width={'85%'}
```

Comment:
the modal has a horizontal scroll at mobile width. Is there something overriding this?